about-me.md

```diff
@@ -6,4 +6,4 @@ Hi, I'm Megan. I'm a strat ops person at the *Defense Digital Service*.
 
 - **Past Companies and Organizations:** Palantir Technologies. US Government. Minnesotan!
 - **Hobbies:** I like board games, dance, and being outside.
-- **Favorite Website:** [WaPo](https://www.washingtonpost.com/)
+- **Favorite Website:** [WaPo](https://www.washingtonpost.com/).  Drew!  Help me be creative?
```

Comment:
This is obviously the best website ever: [Under Construction](http://textfiles.com/underconstruction/)